### PR TITLE
fix: update zoom description

### LINKED ISSUE #918: Add more precision to the existing description of initial zoom selection
For the user, it can be difficult to know the meaning of the value ( ex : a zoom set to 7 is close to or outside the tile ?).

### PR DESCRIPTION
**Pull request by MAudelGisaia**
- close #918

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -447,9 +447,9 @@
 	"Network Fetching Level": "Fetching data",
 	"Network Fetching Level description": "Network Analytics data are fetched on the map according to a X.Y.Z grid. Choose the zoom level Z of the X.Y.Z tiles.",
 	"Zoom min": "Zoom min",
-	"zoom min description": "Minimum zoom level to display the layer",
+	"zoom min description": "Minimum zoom level to display the layer.For example, setting a minimum zoom to 3 will make your layer visible from the continental scale. Before this threshold, it will not be visible.",
```
Comment (QuCMGisaia) [inline src/assets/i18n/en.json:450]: Need a space after the period.

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -446,9 +446,9 @@
 	"Network Fetching Level": "Récupération des données",
 	"Network Fetching Level description": "Les données Network Analytics sont récupérées sur la carte selon une grille X.Y.Z. Choisir le niveau de zoom Z correspondant aux tuiles X.Y.Z",
 	"Zoom min": "Zoom minimal",
-	"zoom min description": "Niveau de zoom minimal pour visualiser la couche dans l'application",
+	"zoom min description": "Niveau de zoom minimal pour visualiser la couche dans l'application.Par exemple, un zoom minimal configurer à 3 rendra votre couche visible à partir de l'échelle continentale. Avant ce seuil, elle ne sera pas visible.",
```
Comment (QuCMGisaia) [inline src/assets/i18n/fr.json:449]: configurer -> configuré

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -446,9 +446,9 @@
...
 	"Zoom max": "Zoom maximal",
-	"zoom max description": "Niveau de zoom maximal pour visualiser la couche dans l'application",
+	"zoom max description": "Niveau de zoom maximal pour visualiser la couche dans l'application. Par exemple, un zoom maximal configurer à 15 rendra votre couche visible jusqu'à l'échelle du bâtiment. Au-delà de ce seuil, elle ne sera plus visible.",
```
Comment (QuCMGisaia) [inline src/assets/i18n/fr.json:451]: configurer -> configuré

**Review by QuCMGisaia — APPROVED** [review]

File: src/assets/i18n/fr.json [hunk 4]
```diff
@@ -359,7 +359,7 @@
 	"Preview saved temporarily. Save the dashboard to validate the preview too.": "L'aperçu est sauvegardé temporairement. Sauvegarder le dashboard pour valider également l'aperçu.",
 	"Do you really want to delete the visualisation set?": "Voulez-vous vraiment supprimer cet espace de visualisation ?",
 	"Initial zoom": "Zoom initial",
-	"Initial zoom description": "Niveau de zoom à l'ouverture de l'application",
+	"Initial zoom description": "Niveau de zoom de la carte lorsque l'application est chargée pour la première fois, 0 étant le niveau de zoom le plus faible (entièrement dézoomé) et 22 le niveau de zoom le plus élevé (entièrement zoomé). Par exemple, au niveau de zoom 0, vous pouvez voir la terre et au niveau de zoom 15, vous pouvez voir les bâtiments.",
```
Comment (MohamedHamouGisaia) [inline src/assets/i18n/fr.json:362]: i suggest to replace 'terre' by 'terre entière'

**Review by MohamedHamouGisaia — APPROVED** [review]

**Comment by MohamedHamouGisaia** [comment]
LGTM ! you can merge

**Pull request merged**